### PR TITLE
add dynamic support for floor/logical_not/sign/round/isinf/isnan

### PR DESCRIPTION
**Pull request by lanluo-nvidia**
# Description

add dynamic support for floor/logical_not/sign/round/isinf/isnan

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: tests/py/dynamo/conversion/test_isinf_aten.py [hunk 1]
```diff
@@ -37,6 +38,50 @@ def forward(self, input):
...
+        inputs = [
+            Input(
+                min_shape=(1, 2, 3),
+                opt_shape=(3, 2, 3),
+                max_shape=(5, 3, 3),
+                dtype=torch.float32,
+                torch_tensor=torch.tensor(
+                    (4.7, -2.3, float("inf")), dtype=torch.float32
+                ).cuda(),
+                is_shape_tensor=True,
```
Comment (peri044) [inline tests/py/dynamo/conversion/test_isinf_aten.py:55]: Why is this true here ?

**Comment by peri044** [comment]
LGTM
